fix(alerts): Add current assignee to issue alert suggested assignees (#7674)
getsentry/sentry-docs: src/docs/product/alerts/create-alerts/issue-alert-config.mdx

### Suggested Assignees

> Suggested Assignees are people or teams that are currently assigned the issue, assigned as [owners of an issue](/product/issues/ownership-rules/) or have been identified as authors of the commit and pull request in which a [suspect commit](/product/issues/suspect-commits/) was made. When an alert is triggered, they can choose to be notified via email or Slack, (depending on their [notification settings](/product/alerts/notifications/notification-settings/).

Reviewer: ```suggestion
Suggested Assignees include people or teams currently assigned the issue, issue owners defined in [ownership rules](/product/issues/ownership-rules/), and anyone who has been identified as author of a [suspect commit](/product/issues/suspect-commits/). When an alert is triggered, they can choose to be notified via email or Slack (depending on their [notification settings](/product/alerts/notifications/notification-settings/)).
```

> If a suggested assignee hasn't been assigned, configured or found, the notification will be sent to the teams or members specified in the fallback notification setting, as shown below:

Reviewer: ```suggestion
If no suggested assignees are found, the notification will be sent to the teams or members specified in the fallback notification setting, as shown below:
```
